tests/test_tmjob.py

```diff
@@ -258,6 +261,14 @@ def test_tm_job_split_angles(self):
         self.assertTrue(np.abs(angle - read_mrc(TEST_ANGLES)).sum() == 0,
                         msg='split rotation search should be identical')
 
+    def test_parallel_breaking(self):
+        try:
+            _ = run_job_parallel(self.job, volume_splits=(1, 2, 1), gpu_ids=[0, -1], mute=True)
+        except RuntimeError:
+            self.assertEqual(len(multiprocessing.active_children()), 0,
+                             msg='a process was still lingering after a parallel job with partially invalid resources '
+                                 'was started')
```

Comment:
please add an `else` in case the error did not get triggered somehow
```suggestion
        except RuntimeError:
            self.assertEqual(len(multiprocessing.active_children()), 0,
                             msg='a process was still lingering after a parallel job with partially invalid resources '
                                 'was started')
        else:
            self.fail('This should have given a RuntimeError')   
```